test_fms/fms2_io/test_fms2_io.sh

```diff
@@ -1,3 +1,12 @@
 #!/bin/sh
-export PATH="$PATH:../bats/bin"
-bats test_fms2_io.bats
+
+# This is part of the GFDL FMS package. This is a shell script to
+# execute tests in the test_fms/field_manager directory.
+
+# Ed Hartnett 11/29/19
```

Comment:
My name should be removed, and the author's substituted, since I did not create this file. ;-)